app/styles/ilios-common/components/new-learningmaterial.scss

```diff
@@ -15,4 +15,29 @@
   .description {
     @include ilios-browser-defaults;
   }
+
```

Comment:
these new styles are inconsistent with the rest of the form. the label should sit above the control, not be to the right of it.

Reply:
I styled it the way it looks now in frontend. With the label above and the button by itself on a line it looked weird.